ccd-definition/ComplexTypes/CareSupervision/1_ManageDocuments/ManageDocumentBundle.json

```diff
@@ -0,0 +1,36 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ManageDocumentBundle",
+    "ListElementCode": "name",
+    "FieldType": "Text",
+    "ElementLabel": "Document name",
+    "HintText": "TODO Try and make this more generic so we can reuse the same object",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ManageDocumentBundle",
+    "ListElementCode": "notes",
+    "FieldType": "TextArea",
+    "ElementLabel": "Notes",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "ManageDocumentBundle",
+    "ListElementCode": "dateTimeReceived",
+    "FieldType": "DateTime",
```

Comment:
Do we need any validation in the UI?

Reply:
added it

Reply:
Where?

Reply:
ahh you mean validation user while entering !! i was missing date assertion before and i thought you were talking about that and added it. 

No validation mentioned in the ticket but i will check with Gail.

Reply:
validation added